overlord/configstate/configcore/export_test.go

```diff
@@ -35,13 +32,18 @@ var (
 	SwitchDisableService = switchDisableService
 	UpdateKeyValueStream = updateKeyValueStream
 	AddFSOnlyHandler     = addFSOnlyHandler
-	AddWithStateHandler  = addWithStateHandler
 	FilesystemOnlyApply  = filesystemOnlyApply
-	StoreReachable       = storeReachable
 )
 
 type PlainCoreConfig = plainCoreConfig
 
+// MockRun is used for tests that run also when nomanagers flag is
+// set, that is, for config groups that do not need access to the
+// state.
+func MockRun(dev sysconfig.Device, cfg ConfGetter) error {
+	return filesystemOnlyRun(dev, cfg, nil)
+}
+
```

Comment:
I this point this could be clear as:

```
var FSOnlyRun = filesystemOnlyRun
```

or Filesystem... if you prefer the longer name.

Reply:
I was too hasty, we need to pass the nil, but the MockRun name is not quite right now because this will only run the filesystem-only handlers, not others, so a different name might be appropriate anyway.